dev/merge_spark_pr.py

```diff
@@ -305,7 +305,9 @@ def resolve_jira_issue(merge_branches, comment, default_jira_id=""):
     versions = [
         x
         for x in versions
-        if not x.raw["released"] and not x.raw["archived"] and re.match(r"\d+\.\d+\.\d+", x.name)
+        if not x.raw["released"]
+        and not x.raw["archived"]
+        and re.match(r"kubernetes-operator-\d+\.\d+\.\d+", x.name)
```

Comment:
Due to the maximum line length limit, I applied Apache Spark's `reformat-python` script. I'll copy it into this repo seperately.
- https://github.com/apache/spark/blob/master/dev/reformat-python